students/hyska_monika/lesson_01_basics/ClockFace1.py

```diff
@@ -0,0 +1,11 @@
+# The program print the angle (in degrees) of the hour hand on the clock
+
+H = int(input("Put hours: "))
+M = int(input("Put minutes: "))
+S = int(input("Put seconds: "))
+
+S_sum = S + M * 60 + H * 3600   # change to seconds
+
+H_st = 30 / 3600 * S_sum    # 1h = 30 degree, 360/12
```

Comment:
The PEP8 (Python code style recommendation) says that you should use UPPER_CASE constants and Capitalization for classes. In general it's always about a balance between `too_long_a_name` and `shrt`. A good compromise is about 1-3 words: `seconds`, `sum_seconds`...